src/game/boe.main.cpp

```diff
@@ -233,11 +234,18 @@ std::string log_file;
 
 bool init_action_log(std::string command, std::string file) {
 	if (command == "record") {
+		// If a filename is given, use it as a base, but insert a timestamp for uniqueness.
+		if (file.empty())
+			file = "BoE";
+
+		if (boost::ends_with(file, ".xml"))
```

Comment:
I think Boost.Algorithm is a new dependency, right? I don't mind adding it, but it needs to be mentioned in the README.

Reply:
It didn't need to be added to the setup of the CI for any of the platforms--it must be a dependency of one of the packages we already install. 

The readme says

> - [Boost](http://www.boost.org/) - Filesystem and System, plus several header-only
>  libraries; if you're picky, you can run scons and see it enumerate exactly which
>  libraries are needed

So I think it could fall under that umbrella.

Reply:
Nice to see that the readme is contradicting itself slightly – the vcpkg instructions _do_ enumerate every necessary Boost library, after all. (Though I'm not confident there are no others missing.)